addons/stock/report/report_stock_rule.xml

```diff
@@ -0,0 +1,170 @@
+                                                            <t t-set="padding" t-value="50.0/acc"/>
+                                                            <div class="o_report_stock_rule_rule_main" t-att-res-id="rule[0].id" t-att-res-model="rule[0]._name" view-type="form" t-attf-title="#{rule[0].route_id.display_name}">
+                                                                <div class="o_report_stock_rule_rule"  t-attf-style="padding-left: #{padding}%; padding-right: #{padding}%;">
+                                                                    <t t-if="rule[1] == 'destination'">
+                                                                        <t t-if="rule[0].procure_method == 'make_to_order'">
+                                                                            <t t-call="stock.report_stock_rule_suspension_points"/>
+                                                                        </t>
+                                                                        <t t-if="rule[0].action in ('push', 'pull_push')">
+                                                                            <t t-call="stock.report_stock_rule_right_arrow"/>
+                                                                        </t>
+                                                                    </t>
+                                                                    <t t-if="rule[1] == 'origin' and rule[0].action in ('pull', 'pull_push')">
+                                                                        <t t-call="stock.report_stock_rule_left_arrow"/>
+                                                                    </t>
+                                                                    <t t-call="stock.report_stock_rule_line"/>
+                                                                    <t t-if="rule[1] == 'destination' and rule[0].action in ('pull', 'pull_push')">
+                                                                        <t t-call="stock.report_stock_rule_right_arrow"/>
+                                                                    </t>
+                                                                    <t t-if="rule[1] == 'origin'">
+                                                                        <t t-if="rule[0].action in ('push', 'pull_push')">
+                                                                            <t t-call="stock.report_stock_rule_left_arrow"/>
+                                                                        </t>
+                                                                        <t t-if="rule[0].procure_method == 'make_to_order'">
+                                                                            <t t-call="stock.report_stock_rule_suspension_points"/>
+                                                                        </t>
+                                                                    </t>
```

Comment:
maybe all these t-if t-call could be replaced by using something like
t-att-call="'stock.report_stock_rule_%s_%s_%s' % (rule[1], rule[0].procure_method, rule[0].procure_method)"
not sure if t-att or t-attf- is supported or if we'd need <t t-call="main_template"><t t-set='var' t-value='machin'> and some if to call the right template.. let's discuss tomorrow if you don't get it